app/js/event/event.sort.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('upcomingStops.event')
+    .factory('eventFilter', eventFilter);
```

Comment:
Thanks for renaming the file :+1: 
Could you also rename all the eventFilter to EventSort to reflect this change?